tikv/raftstore/peer_storage.go

```diff
@@ -291,6 +291,37 @@ func getMsg(engine *badger.DB, key []byte, msg proto.Message) error {
 	return proto.Unmarshal(val, msg)
 }
 
+type storageError string
+
+func (e storageError) Error() string {
+	return string(e)
+}
+
+func getRegionLocalStateMsg(db *badger.DB, regionId uint64) (*rspb.RegionLocalState, error) {
```

Comment:
We can omit `Msg` for the function name.

Reply:
yes